avocado/core/test.py

```diff
@@ -416,7 +416,7 @@ def get_state(self):
             self._update_time_elapsed()
         preserve_attr = ['basedir', 'debugdir', 'depsdir', 'fail_reason',
                          'logdir', 'logfile', 'name', 'resultsdir', 'srcdir',
-                         'status', 'sysinfodir', 'text_output', 'time_elapsed',
```

Comment:
It was available so we published it in `get_status`. But inside our code we do not query for it and I seriously doubt anyone is using `get_status` to get `sysinfodir` from plugins/test itself.